languages/storefront-footer-bar.pot

```diff
@@ -1,51 +1,52 @@
-# Copyright (C) 2015 Storefront Footer Bar
+# Copyright (C) 2020 WooThemes
```

Comment:
Lets update the plugin header info while we are at

Reply:
Fixed 😀